### PR TITLE
[DET-2972] Fix cifar10_cnn_pytorch bugs and add a convergence test

### PR DESCRIPTION
**Pull request by yoavz**
The convergence test will run `cifar10_cnn_pytorch` for 500 steps nightly and verify the validation accuracy is above 74.5%. 

The test experiment I ran takes about ~10 minutes to finish 500 steps and reaches a best validation accuracy of ~74.9%. I set the target threshold accuracy at slightly below that to give some slack for non-deterministic factors. We might need to revisit this based on how flaky this threshold ends up being.

### CODE REVIEW

File: examples/official/cifar10_cnn_pytorch/const.yaml [hunk 1]
```diff
@@ -11,6 +13,7 @@ hyperparameters:
 searcher:
   name: single
   metric: validation_error
-  max_steps: 50
+  max_steps: 500
```
Comment (katport) [inline examples/official/cifar10_cnn_pytorch/const.yaml:16]: we may want to add how many epochs this equates to - similar to the mnist.

File: tests/integrations/test_convergence.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from tests.integrations import config as conf
+from tests.integrations import experiment as exp
+
+
+@pytest.mark.nightly  # type: ignore
+def test_cifar10_pytorch_accuracy() -> None:
+    config = conf.load_config(conf.official_examples_path("cifar10_cnn_pytorch/const.yaml"))
+    config = conf.set_max_steps(config, 2)
```
Comment (brain-good) [inline tests/integrations/test_convergence.py:10]: issue: don't think we want to do this.. we should be directly testing on `conf.official_examples_path("cifar10_cnn_pytorch/const.yaml")` right?

Comment (yoavz) [reply tests/integrations/test_convergence.py]: Good catch -- accidentally left this step modification from testing. removed.

File: examples/official/cifar10_cnn_pytorch/const.yaml (hunk 1 above)
Comment (yoavz) [reply examples/official/cifar10_cnn_pytorch/const.yaml]: added

**Pull request merged**